authentication/auth.go

```diff
@@ -73,8 +73,8 @@ func VerifyLogin(clientID string) (string, error) {
 		ClientSecret: "",
 		Scopes:       []string{"openid"},
 		Endpoint: oauth2.Endpoint{
-			AuthURL:  "https://sso.stage.redhat.com/auth/realms/redhat-external/protocol/openid-connect/auth",
-			TokenURL: "https://sso.stage.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token",
+			AuthURL:  "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/auth",
+			TokenURL: "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token",
```

Comment:
Let's use the SSO constant from `authentication/transport_wrapper.go`. And create new constants for Redirect URL and Port (shared with the serve function)